doc/contributor/howto-guide-adding-generated-libraries.md

```diff
@@ -197,6 +219,15 @@ arguments in the CI builds.
 
 - `google/cloud/${library}/CMakeLists.txt`
 
+The scaffold generator does not know the current version of `google-cloud-cpp`.
+We can copy it from a library that should be up-to-date.
+
+```sh
+sed -i '/workspace(/q' google/cloud/${library}/quickstart/WORKSPACE.bazel
```

Comment:
For a future PR, it might be possible to create a `generator/internal/WORKSPACE.bazel` file which gets automatically updated by the robots.  Then we would need to read the file from the scaffold generator, which requires some magic with Bazel.